obspy/core/util/attribdict.py

```diff
@@ -182,7 +184,8 @@ def _cast_type(self, key, value):
         :return: value cast to correct type.
         """
         typ = self._types[key]
-        new_type = typ[0] if isinstance(typ, collections.Sequence) else typ
+        new_type = typ[0] if isinstance(
+            typ, compatibility.collections_abc.Sequence) else typ
```

Comment:
As the function call is not the first thing on the line, I don't really like this break style and would rather wrap everything in parentheses. But since this is going to be reverted soon, I'm not really going to hold up on it.

Reply:
Fair enough. I'll change it. I think we should adapt `black` soon in any case. It will just make rebasing any PR a nightmare 🙈 

But not having to worry about formatting anymore is just so good.